src/main/java/io/github/nucleuspowered/nucleus/modules/mute/commands/MuteCommand.java

```diff
@@ -63,6 +65,13 @@
     }
 
     @Override
+    public Map<String, PermissionInformation> permissionSuffixesToRegister() {
+        Map<String, PermissionInformation> m = new HashMap<>();
+        m.put("bypass", new PermissionInformation(Util.getMessageWithFormat("permission.mute.bypass"), SuggestedLevel.MOD));
```

Comment:
Missed this one `exempt.length` please (and, while we're at it, may as well add the `exempt.target` permission)